test/test_cuda.py

```diff
@@ -3259,6 +3260,18 @@ def test_graph_capture_simple(self):
 
         self.assertTrue(b.sum().item() == 11000.)
 
+    @unittest.skipIf((not TEST_CUDA) or
+                     TEST_WITH_ROCM or
+                     int(torch.version.cuda.split(".")[0]) < 11, "CUDA >= 11.0 required for graphs")
+    def test_graph_warn_if_has_zero_nodes(self):
+        with warnings.catch_warnings(record=True) as caught:
```

Comment:
The test is having issues with catching the warnings. @jjsjann123 pointed out that the possible cause is call_guard in pybing11  https://github.com/pytorch/pytorch/blob/master/torch/csrc/cuda/Graph.cpp#L31-L39 

Any ideas on how to catch this warning?
cc @ezyang @ngimel

Reply:
You need the wrapper that clears the warning queue into Python. `HANDLE_TH_ERRORS` and `END_HANDLE_TH_ERRORS_PYBIND` check torch/csrc/README.md

cc @peterbell10

Reply:
I was going to suggest `torch::wrap_pybind_function` but it didn't work with member functions, so I've added that support and wrapped the CUDA graph functions in #88932. So, if you rebase it should start catching the warnings.

Reply:
Thanks @peterbell10, 
The test is passing!